internal/api/api.go

```diff
@@ -76,6 +77,33 @@ func (a *snykApiClient) GetDefaultOrgId() (string, error) {
 	return userInfo.Data.Attributes.DefaultOrgContext, nil
 }
 
+func (a *snykApiClient) GetFeatureFlag(flagname string) (bool, error) {
+	const defaultResult = false
+
+	url := a.url + "/v1/cli-config/feature-flags/" + flagname
+	res, err := a.client.Get(url)
+	if err != nil {
+		return defaultResult, fmt.Errorf("unable to retrieve feature flag: %w", err)
+	}
+	defer res.Body.Close()
+
+	body, err := io.ReadAll(res.Body)
+	if err != nil {
+		return defaultResult, fmt.Errorf("unable to retrieve feature flag: %w", err)
+	}
+
+	var flag contract.OrgFeatureFlagResponse
+	if err = json.Unmarshal(body, &flag); err != nil {
+		return defaultResult, fmt.Errorf("unable to retrieve feature flag (status: %d): %w", res.StatusCode, err)
+	}
+
+	if res.StatusCode != http.StatusOK || flag.Code == "401" || flag.Code == "403" {
+		return defaultResult, nil
+	}
+
+	return true, nil
+}
+
```

Comment:
**praise:** 🥳 Great to see this added as a general helper, I look forward to all of the features we can see roll out across the CLI the now